### PR TITLE
OCP 4.13 : Assisted Installer updates for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s) : 4.13+

Issue : https://issues.redhat.com/browse/MULTIARCH-3153

Link to docs preview : https://58317--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_on_prem_assisted/installing-on-prem-assisted.html

QE review:
- [ ] QE has approved this change.

cc : @SNiemann15

### CODE REVIEW

**Comment by AmadeusPodvratnik** [comment]
@alishaIBM Looks good to me.

File: installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc [hunk 1]
```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports both x86-64 and AArch64 CPU architectures.
+You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports x86-64, AArch64, ppc64le, s390x and arm64 CPU architectures.
+
```
Comment (manojnkumar) [inline installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc:10]: Are AArch64 and arm64 CPU architectures considered distinct?

Comment (SNiemann15) [reply installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc:10]: @kelbrown20 Do you know when to use arm64 and when AArch64? I asume it's the same but we should probably only have the architecture label for ARM once.

**Comment by holgwolf** [comment]
LGTM

File: installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc (hunk 1 above)
Comment (alishaIBM) [reply installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc:10]: Thanks @kelbrown20. Updated as mentioned above.

File: installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc [hunk 2]
```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports both x86-64 and AArch64 CPU architectures.
+You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports x86_64, AArch64, ppc64le, and s390x CPU architectures.
+
+[NOTE]
+====
+Currently installing {product-title} on {ibmzProductName} (s390x) is only supported with RHEL KVM installations.
```
Comment (mburke5678) [inline installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc:13]: You can probably remove the _Currently_. 

s/only supported/supported only

File: modules/assisted-installer-using-the-assisted-installer.adoc [hunk 3]
```diff
@@ -35,6 +35,10 @@ The {ai-full} provides installation functionality as a service. This software-as
 The {ai-full} supports installing {product-title} on premises in a connected environment, including with an optional HTTP/S proxy. It can install the following:
 
 * Highly available {product-title} or Single Node OpenShift (SNO)
+[NOTE]
+====
+Currently, SNO is not supported on {ibmzProductName} and {ibmpowerProductName}.
```
Comment (mburke5678) [inline modules/assisted-installer-using-the-assisted-installer.adoc:40]: You can probably remove the Currently.

Comment (mburke5678) [reply modules/assisted-installer-using-the-assisted-installer.adoc:40]: I think you need `+` characters before and after the admonition.

```
+
[NOTE]
====
Currently, SNO is not supported on {ibmzProductName} and {ibmpowerProductName}.
====
+
```

**Comment by mburke5678** [comment]
@alishaIBM A couple of style nits and one formatting issue. Otherwise LGTM

File: installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc (hunk 2 above)
Comment (alishaIBM) [reply installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc:13]: @mburke5678 thanks.

Updated as : 
```
Installing {product-title} on {ibmzProductName} (s390x) is supported only with RHEL KVM installations.
```

File: modules/assisted-installer-using-the-assisted-installer.adoc (hunk 3 above)
Comment (alishaIBM) [reply modules/assisted-installer-using-the-assisted-installer.adoc:40]: @mburke5678 thanks.

Updated as : 
```
+
[NOTE]
====
SNO is not supported on {ibmzProductName} and {ibmpowerProductName}.
====
+
```

**Comment by alishaIBM** [comment]
https://gitlab.cee.redhat.com/telco-team-documentation/assisted-installer-saas-documentation/-/merge_requests/54 is merged.

@opayne1 request you to take a look ( merge review ). Thanks.
cc : @SNiemann15

**Pull request merged**